packages/datadog-instrumentations/src/http2/server.js

```diff
@@ -5,14 +5,15 @@
 
 const {
   channel,
-  addHook,
-  AsyncResource
+  addHook
 } = require('../helpers/instrument')
 const shimmer = require('../../../datadog-shimmer')
 
 const startServerCh = channel('apm:http2:server:request:start')
 const errorServerCh = channel('apm:http2:server:request:error')
 const finishServerCh = channel('apm:http2:server:request:finish')
+// this channel is for wrapping the response emit method and handling store context, it doesn't have any subscribers
```

Comment:
This needs a subscriber to have any effect.

Reply:
Fixed in our review session